module/zfs/vdev.c

```diff
@@ -28,6 +28,7 @@
  * Copyright 2017 Joyent, Inc.
  * Copyright (c) 2017, Intel Corporation.
  * Copyright (c) 2019, Datto Inc. All rights reserved.
+ * Copyright [2021] Hewlett Packard Enterprise Development LP
```

Comment:
Can this be changed to conform with the "standard" syntax?:
Copyright (r) 2021 Hewlett Packard Enterprise Development LP

Reply:
See response to other copyright change request.